frontend/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
```

Comment:
What are we doing for trailing commas with this? In looking at it a little bit more, I think that I might have been convinced that my hereto dislike of trailing commas at the end of lists and arrays might be unwarranted. Might be preferable even as it'd give us nicer git diffs and also would making extending lists easier (I've not enjoyed all the JSON work with the L10N files where I need to add the comma myself all the time). How do you feel about enforcing trailing comma addition?

Reply:
I don't have a strong opinion on this. Personally i do not write trailing commas and let the formatter take care of it. My default setting in VScode was also "es5".

Just checked the docs for the default and had changed:

> Default value changed from es5 to all in v3.0.0

I am happy with any of the three options.

Reply:
So as of now is it setup to add the last comma to it, @to-sta? If not, can that be swapped in a commit? Happy to merge once that's done :) :)

Reply:
Yeah, it will add a trailing comma.

Reply:
Thanks, @to-sta :) Has been a long day of traveling and just wanted to check :)